IdentityCore/src/oauth2/token/MSIDPrimaryRefreshToken.h

```diff
@@ -24,6 +24,14 @@
 #import "MSIDLegacyRefreshToken.h"
 #import "MSIDLegacyCredentialCacheCompatible.h"
 
+typedef NS_ENUM(NSInteger, MSIDExternalPRTKeyLocationType)
+{
+    MSIDExternalPRTKeyLocationTypeNone = 0, // Key isn't stored externally
+    MSIDExternalPRTKeyLocationTypeWPJ = 1, // WPJ device and STK keys stored in secure enclave are used for PRT operations (for device-bound cases)
```

Comment:
can we name this MSIDExternalPRTKeyLocationTypeWPJSecureEnclave ? as it wil also convey WPJed and using SE

Reply:
Sure

Reply:
Addressed, please take another look